plugins/wazuh-fleet/public/components/agents/details/dashboard/events-count.tsx

```diff
@@ -1,51 +1,28 @@
 import React from 'react';
 import { getPlugins } from '../../../../plugin-services';
-import { getDashboardPanels } from './dashboard_panels';
 import { ViewMode } from '../../../../../../../src/plugins/embeddable/public';
-import {
-  EuiPanel,
-  EuiFlexItem,
-  EuiFlexGroup,
-  EuiSpacer,
-  EuiText,
-} from '@elastic/eui';
+import { useTimeFilter } from '../../../common/table-indexer/components/search-bar/hooks/use-time-filter';
+import { LoadingSpinner } from '../../../common/loading-spinner/loading-spinner';
+import { getDashboardPanels } from './dashboard_panels';
 
 export interface EventsCountProps {
-  AlertsDataSource: any;
-  AlertsDataSourceRepository: any;
-  useDataSource: any;
-  useTimeFilter: any;
-  LoadingSpinner: any;
+  indexPattern: any;
+  filters: any[];
```

Comment:
The filters prop is typed as any. Using any defeats the purpose of TypeScript. Consider defining a specific type for filters to improve type safety and maintainability.

import { Filter } from '../../../../src/plugins/data/public';

```suggestion
  filters: Filter[];
```

Reply:
Solved https://github.com/wazuh/wazuh-dashboard-plugins/commit/fd1f25fda8819043e513dd6a9be6b1f49d868c81